lib/id3c/cli/command/etl/presence_absence.py

```diff
@@ -285,9 +285,15 @@ def sample_identifier(db: DatabaseSession, barcode: str) -> Optional[str]:
     """
     identifier = find_identifier(db, barcode)
 
+    valid_identifiers = [
+        "samples",
+        "collections-uw-tiny-swabs",
```

Comment:
I think we want to leave out `collections-uw-tiny-swabs` because it should only be used for the pilot, and marked experimental/ignored by this ETL.

Reply:
OK, I'll fix that so it doesn't get used.